packages/core/src/server/index.ts

```diff
@@ -68,9 +70,10 @@ function getFinalAPISchema() {
 export const getEnvelop = async () =>
   envelop({
     plugins: [
+      useEngine(GraphQLJS),
```

Comment:
Ref: https://the-guild.dev/graphql/envelop/v3/guides/migrating-from-v2-to-v3#1-remove-graphql-as-a-peer-dependency